src/Symfony/Component/Validator/Constraints/ExpressionLanguageSyntaxValidator.php

```diff
@@ -43,7 +43,7 @@ public function validate($expression, Constraint $constraint): void
         }
 
         try {
-            $this->expressionLanguage->lint($expression, ($constraint->validateNames ? ($constraint->names ?? []) : null));
+            $this->getExpressionLanguage()->lint($expression, ($constraint->validateNames ? ($constraint->names ?? []) : null));
```

Comment:
What if we do the "check" in the method without defining a new method (as it's only used once?) ?

Reply:
Good idea, that wasn't needed.
Removed.